PowerEditor/Test/FunctionList/python/unitTest

```diff
@@ -277,4 +277,4 @@ def krusk(E_and_n):
                     break
                 s[j].update(s[i])
                 s.pop(i)
-                break
```

Comment:
Though not documented as such the "no new line at end of file" might be intended for testing proper operation of the Python Function List parser.

Reply:
Oh, did I mess anything at the end of the file? I didn't change it though, I just edited the words in the comments, and that too in github web editor. What should I do to fix it?